tools/mob_suite/mob_typer.xml

```diff
@@ -1,153 +1,113 @@
-<tool id="mob_typer" name="MOB-Typer" version="2.0.5+galaxy0">
+<tool id="mob_typer" name="MOB-Typer" version="3.0.0">
   <description>Get the plasmid type and mobility given its sequence</description>
   <requirements>
-     <requirement type="package" version="2.0.5">mob_suite</requirement>
+     <requirement type="package" version="3.0.0">mob_suite</requirement>
   </requirements>
   <version_command>mob_typer --version</version_command>
   <command detect_errors="exit_code">
   <![CDATA[
     ln -s '${input}' '${input.name}' &&
     mob_typer  --num_threads \${GALAXY_SLOTS:-4} --infile '${input.name}'
-    ${host_range_detailed}
+
+
+   #if $adv_param.multi
+    --multi
+   #end if 
+
    --min_rep_evalue '$adv_param.min_rep_evalue_value'
    --min_mob_evalue '$adv_param.min_mob_evalue_value'
    --min_con_evalue '$adv_param.min_con_evalue_value'
-   --min_ori_evalue '$adv_param.min_ori_evalue_value'
-   --min_mpf_evalue '$adv_param.min_mpf_evalue'
+   --min_length '$adv_param.min_length'
    --min_rep_ident '$adv_param.min_rep_ident'
    --min_mob_ident '$adv_param.min_mob_ident'
-   --min_ori_ident '$adv_param.min_ori_ident'
-   --min_mpf_ident '$adv_param.min_mpf_ident'
-    --outdir 'outdir';
+
+   --min_con_ident '$adv_param.min_con_ident'
+   --min_rep_cov '$adv_param.min_rep_cov'
+   --min_mob_cov '$adv_param.min_mob_cov'
+   --min_con_cov '$adv_param.min_con_cov'
+   --min_overlap '$adv_param.min_overlap'
+   
+   #if $adv_param.plasmid_db
+   --plasmid_db $adv_param.plasmid_db
+   #end if
+
+   #if $adv_param.plasmid_db
+   --plasmid_db $adv_param.plasmid_db
+   #end if
+
+   #if $adv_param.plasmid_mash_db
+   --plasmid_mash_db '$adv_param.plasmid_mash_db'
+   #end if
+
+   #if $adv_param.plasmid_meta
+   --plasmid_meta '$adv_param.plasmid_meta'
+   #end if
+
+   #if $adv_param.repetitive_mask
+   --repetitive_mask '$adv_param.repetitive_mask'
+   #end if
+
+   #if $adv_param.plasmid_mob
+   --plasmid_mob $adv_param.plasmid_mob
```

Comment:
Is this a freeform field? If so surround with `'`

Reply:
Done. Thank you